src/MicrosoftTeams.ts

```diff
@@ -76,7 +80,7 @@ namespace microsoftTeams {
     remove: "remove"
   };
 
-  export enum HostClientTypes {
+  export const enum HostClientType {
```

Comment:
If app developer who is using HostClientTypes need to update teams js version then they need to update their code. should we keep name same ?

Reply:
Yeah, but we just added this with v1.3 so I think it's still safe to fix it. I would rather do that now before too many folks have taken a dependency on the old name.